pyproject.toml

```diff
@@ -22,6 +22,7 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "pydantic >= 2.0",
+    'tomli; python_version < "3.11"',
```

Comment:
Yeah, but the compat-file keeps becoming harder to test :)

Reply:
Actually, all of the compats so far are things that are available in python 3.11 so I guess instead of attempting to import, we could do a python version test. No, I will not demand use of Python 3.11 for this *library* because it is used by curitz on debian oldstable.